### PR TITLE
Refactor `/patients/patient/patient.js` tests for jsonapi

### PR DESCRIPTION
**Pull request by nmajor25**
Shortcut Story ID: [sc-49487]

### CODE REVIEW

File: test/integration/patients/patient/patient.js [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
   specify('context trail', function() {
     cy
       .routesForPatientDashboard()
       .routeActions()
       .routePatient(fx => {
-        fx.data.id = '1';
-        fx.data.attributes.first_name = 'First';
-        fx.data.attributes.last_name = 'Last';
+        fx.data = getPatient({
+          id: testPatientId,
```
Comment (paulfalgout) [inline test/integration/patients/patient/patient.js]: I don't know that an id needs to be set?  `getPatient` will return an id that the visit can then use.

**Review by paulfalgout — CHANGES_REQUESTED** [review]
I think we can remove the `uuid` and just use the ids returned from` getFoo` for the `visits`

File: test/integration/patients/patient/patient.js (hunk 1 above)
Comment (nmajor25) [reply test/integration/patients/patient/patient.js]: I think that makes sense as well. I just pushed an updated commit...

Comment (nmajor25) [reply test/integration/patients/patient/patient.js]: I'll work on making these same types of changes to the other PRs as well.

**Comment by nmajor25** [comment]
@paulfalgout I think this PR is good to go? Unless I'm missing something else that needs to be changed...

**Review by paulfalgout — APPROVED** [review]

**Pull request merged**